### PR TITLE
Add Multi-Account Containers integration as a feature

### LINKED ISSUE #2326: Add conditional "Learn more" secondary link to VPNFeatureTour {}
The What's New panel for the Multi-Account Container integration includes a "Learn more" link. We need to update VPNFeatureTour {} to conditionally show a secondary link. 

**Figma spec:**
https://www.figma.com/file/ziJ5tGCqhfst4svjtKJlbr/2.7--Feature-Enhancements?node-id=336%3A7707

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/VPN-1372)

### LINKED ISSUE #2327: Add MAC integration as a feature and add MAC 'What's new' content
**Figma:**
https://www.figma.com/file/ziJ5tGCqhfst4svjtKJlbr/2.7--Feature-Enhancements?node-id=336%3A7707

Summoning [~accountid:6047c4eeaddbcd006ba87be8] or [~accountid:60482eff2b3f9a006a3b70f5] for help with story pointing here.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/VPN-1373)

### PR DESCRIPTION
**Pull request by flozia**
This PR adds Multi-Accounts Containers as a feature to the VPN features list so we can show it in our ”What’s new” tour for desktop. Additionally there is now a new property for linking additional external information for a feature.

**Design**
https://www.figma.com/file/ziJ5tGCqhfst4svjtKJlbr/2.7--Feature-Enhancements?node-id=336%3A7707

**ToDos**
- [x] Add Multi-Account Containers to features list
- [x] Resources for ”What’s new”
- [x] Optional “Learn more” link
- [x] UTM params
- [x] Add final strings

**Linked issues**
- Closes #2327
- Closes #2326

### CODE REVIEW

**Review by strseb — APPROVED** [review]
Nice!

**Comment by lesleyjanenorton** [comment]
@chrisrooneys I remember talking about this but I can't remember where we left everything with the strings for this. For reference, it's this string that I think might seem confusing to people who haven't heard of Multi-Account Containers and don't know what 'containers' means in this context. 

Screen Shot 2021-12-06 at 9 02 21 AM

**Comment by chrisrooneys** [comment]
image
And updated in spec: https://www.figma.com/file/ziJ5tGCqhfst4svjtKJlbr/2.7--Feature-Enhancements?node-id=317%3A10225

**Review by lesleyjanenorton — APPROVED** [review]

File: translations/strings.yaml [hunk 1]
```diff
@@ -220,6 +220,10 @@ whatsNewReleaseNotes:
   tourSystemNotifHeader: What’s new in VPN
   tourSystemNotifBody: Check out all the new features available in the latest version of Mozilla VPN!
   tourSystemNotifCTA: Take the tour
+  # Multi-account containers
+  multiAccountContainersHeader: Mozilla VPN in Firefox
+  multiAccountContainersShortDescription: Mozilla VPN is now integrated with the Multi-Account Containers add-on for more privacy and security.
```
Comment (flodolo) [inline translations/strings.yaml:225]: Please add a note that `Multi-Account Containers` is the name of the add-on, and shouldn't be translated.

**Review by flodolo — APPROVED** [review]

**Pull request merged**